src/components/Commands/FacetedFilter.tsx

```diff
@@ -53,10 +54,17 @@ export const FacetedFilter = <TData, TValue>({
           >
             <CheckIcon size={18} strokeWidth={1.75} />
           </div>
-          {option.icon && (
-            <option.icon size={18} strokeWidth={1.75} />
-          )}
-          <span>{option.label}</span>
+          <div className='flex items-center gap-2'>
+            {option.icon && (
+              <option.icon
+                size={18}
+                strokeWidth={1.75}
+                color={DocumentStatuses.find((status) => status.value === option.value)?.iconProps?.color}
+                className={DocumentStatuses.find((status) => status.value === option.value)?.iconProps?.className}
+              />
```

Comment:
This is a generic component, using `DocumentStatuses` here works now since we're not using colors for any other filter option.

I think we can just do:
```suggestion
              <option.icon
                size={18}
                strokeWidth={1.75}
                color={option.iconProps?.color}
                className={option.iconProps?.className}
              />
```
Since `options` should be declared on the column definition. As defined here: https://github.com/ttab/elephant-chrome/pull/1708/files#diff-c85496fd9266e099f4b6aaea3a4d7c2716dbde1d0782840b89ea7af91bae718dR38